app/views/layouts/_expiration_notice.html.erb

```diff
@@ -5,17 +5,29 @@
   <div class="expiration-content">
     <div class="expiration-head-panel">
       <h1 class="expiration-title"><%= t("expiration.title") %></h1>
-      <h2 class="expiration-subtitle" ><%= t("expiration.sub_title") %></h2>
+      <h2 class="expiration-subtitle" >
+        <% if feature_enabled?(:new_plan_page) %><%= t("expiration.sub_title_new") %>
+        <% else %><%= t("expiration.sub_title") %>
+        <% end %>
+      </h2>
+      <% if feature_enabled?(:new_plan_page) %>
```

Comment:
It shouldn't be displayed at all then, see https://github.com/sharetribe/sharetribe/pull/1481/files#diff-127a850b7d7b20173ad3fba88f8a25e3R779